package.json

```diff
@@ -58,7 +58,7 @@
     "react-native-notifications": "1.2.6",
     "react-native-optimized-flatlist": "^1.0.4",
     "react-native-orientation-locker": "^1.1.5",
-    "react-native-picker-select": "^5.2.3",
+    "react-native-picker-select": "^6.0.0",
```

Comment:
Current `develop` branch is on beta and working under `6.3.0`.
Can you merge and try again?